storagenode/peer.go

```diff
@@ -354,9 +354,9 @@ func (peer *Peer) Run(ctx context.Context) (err error) {
 		return errs2.IgnoreCanceled(peer.Vouchers.Run(ctx))
 	})
 
-	group.Go(func() error {
-		return errs2.IgnoreCanceled(peer.DB.Bandwidth().Run(ctx))
-	})
+	//group.Go(func() error {
```

Comment:
As @navillasa commented without a `TODO` comment this code have a high change to be deleted by someone introducing changes.

The same a apply for the rest of changes of this branch.